docs/sync-rules/ONEDRIVE.md

```diff
@@ -0,0 +1,57 @@
+### Setting up the OneDrive connector
+
+See the [Developer guide](../../docs/DEVELOPING.md) for setting up connectors.
+
+### Overview
+
+The advanced sync rules consists of a set of rules where each rule can have one or more parameters out of these:
+
+1. userMailAccounts: list of user emails for which the rule would execute.
+2. skipFilesWithExtensions: list of file extensions to be skipped for the sync.
+3. parentPathPattern: glob pattern specifying the path where the files to be synced are present. 
+
+### Example advanced sync rules
+
+#### Advanced rule for indexing the files and folders excluding .xlsx and .docx files
+
+```json
+[
+  {
+    "skipFilesWithExtensions": [".xlsx" , ".docx"]
+  }
+]
+```
+
+#### Advanced rule for indexing the files and folders for user 1 and user 2, excluding .py files
+
+```json
+[
+  {
+    "userMailAccounts": ["user1-domain@onmicrosoft.com", "user2-domain@onmicrosoft.com"],
+    "skipFilesWithExtensions": [".py"]
+  }
+]
+```
```

Comment:
So this is an `AND`, and below is `OR`:

```json
[
  {
    "userMailAccounts": ["user1-domain@onmicrosoft.com", "user2-domain@onmicrosoft.com"]
  },
  {
    "skipFilesWithExtensions": [".py"]
  }
]
```

If so, I think it's worth adding more examples to illustrate `OR` rules

Reply:
Sorry for misunderstanding. I'll add more examples and make it clear. 
In the example you provided, the second rule does not have  `userMailAccounts ` parameter, so the rule would execute for all users and fetch content of all files except content of `.py` files